bentoml/frameworks/keras.py

```diff
@@ -191,8 +187,11 @@ def load(self, path):
                         "Failed to import '{}' module when loading saved "
                         "KerasModelArtifact".format(keras_module_name)
                     )
-
-        self.create_session()
+        else:
```

Comment:
side note: the `else` branch was left unhandled for back-ward compatibility reason when this was first created. But I think it is fine raising this exception here now since this was introduced over a year ago.